Hw1Task2/Hw1Task2/TaxFreeForm.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hw1Task2
+{
+    class TaxFreeForm
+    {
+        public string PassporNumber { get; set; }
```

Comment:
Where do you write your code? Why formatting is so bad?

Reply:
Visual Studio 2017